src/test/java/Homework19.java

```diff
@@ -7,8 +7,6 @@ public class Homework19 extends KoelTesting{
 
     @Test
     public void deletePlaylist() throws InterruptedException {
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

![image](https://user-images.githubusercontent.com/104761482/212462462-4aef60c4-23e3-44d2-a2a3-626c8619b085.png)

![image](https://user-images.githubusercontent.com/104761482/212462496-c725ea0f-3747-4247-9751-e58677b22bde.png)

![image](https://user-images.githubusercontent.com/104761482/212462510-7c0accba-de6d-4081-87af-13bcd60f6c80.png)